### PR TITLE
DAMIEN-242, DAMIEN-243: admin can toggle the locked state of current term

### PR DESCRIPTION
**Pull request by lyttam**
https://jira-secure.berkeley.edu/browse/DAMIEN-242
https://jira-secure.berkeley.edu/browse/DAMIEN-243
https://jira-secure.berkeley.edu/browse/DAMIEN-278

### CODE REVIEW

File: src/views/StatusBoard.vue [hunk 1]
```diff
@@ -196,6 +219,17 @@ export default {
...
       })
     },
+    toggleCurrentTermLocked() {
+      if (this.isCurrentTermLocked) {
+        lockEvaluationTerm(this.$config.currentTermId).then(() => {
+          this.alertScreenReader(`Locked ${this.$config.currentTermName}`)
+        })
+      } else {
+        unlockEvaluationTerm(this.$config.currentTermId).then(() => {
+          this.alertScreenReader(`Unocked ${this.$config.currentTermName}`)
```
Comment (pauline2k) [inline src/views/StatusBoard.vue]: @lyttam New word for the screenreader! Everything else looks good.

**Comment by lyttam** [comment]
Please hold off on merging; I need to clean up the git mess I just made.

**Comment by pauline2k** [comment]
Looks good from here, @lyttam - any cleanup still needed on your end?

**Comment by lyttam** [comment]
Fixed and ready to merge. @pauline2k @cesarvh

**Pull request merged**